.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text=auto
```

Comment:
Is this the default behaviour? And does it mean that all other files in the repo will have windows line endings if checkout out on Windows? Does that _just work_ everywhere else?

Reply:
Ugh. Are you worried about this at all:

> We had a repo with * text=auto, and Git guessed wrong for an image file that it was a text file, causing it to corrupt it as it replaced CR + LF bytes with LF bytes in the object database. That was not a fun one to debug.

I don't _think_ I am for this repo. But it is annoyingly unclear. I still this we should try this though, and hopefully everything will work fine 😬.

I can imagine things being difficult if e.g. people are editing code using a tool that adds CRLF but then the app inside our container can't handle that. (although I suspect it would be able to...)